rasa_core/policies/policy.py

```diff
@@ -29,6 +29,16 @@ def _create_featurizer(cls, featurizer=None):
         else:
             return cls._standard_featurizer()
 
+    @classmethod
+    def _set_priority(cls, priority=None):
+        if priority:
+            return copy.deepcopy(priority)
+        else:
+            logger.warning("Policy should have a default priority defined " +
```

Comment:
I think the default should be `1` and not `0` - any policy that does not set a priority should be treated like keras and embedding policies (this will also make this somewhat more backwards compatible, as this is how we treated them before)

Reply:
I don't even think we should have this warning. The policy base class should set it to a sensible default, if the user writes his custom policy I don't think we need to force them to define a priority.